alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/config/field/SelectConfigField.java

```diff
@@ -36,92 +36,124 @@
     private Collection<LabelValueSelectOption> options;
     private boolean searchable;
     private boolean multiSelect;
+    private boolean removeSelected;
```

Comment:
Will there ever be a case where we don't want to do this?

Reply:
Right now I don't see a reason not to always do this for our product.